source/retryable-reads/tests/README.rst

```diff
@@ -78,8 +78,8 @@ Each YAML file has the following keys:
   - ``failPoint``: Optional, a server failpoint to enable expressed as the
     configureFailPoint command to run on the admin database.
```

Comment:
Can we remove the SERVER-34943 comment now too? I think 4.1.7 is out.

Reply:
Removed!